medmodels/matching/metrics.py

```diff
@@ -109,6 +111,8 @@ def mahalanobis_metric(
     return float(math.sqrt(distance_squared))
 
 
+Metric: TypeAlias = Literal["absolute", "exact", "mahalanobis"]
```

Comment:
Static variables and TypeAliases to the top of the module as per one of the other comments

Reply:
the static part also does not work in this case